### DISCUSSION TITLE
many reverse aliases are autocreated when subscribed to a newsletter that use dynamic reply-to headers for its posts

### DISCUSSION THREAD

**Discussion by lestephane** [simple-login/app#1471] [Q&A]
I'm subscribed to a newsletter that uses a different reply to address per post....
So an additional reverse alias is created each time I receive a post from them.

Is it OK for so many reverse aliases to be created that will never be used again (I never reply to a newsletter post)?

I would like to either avoid creating those automatically (they're based on the reply-to header), or to delete
the unused reverse aliases after some time (if 0 forwards and 0 sent for x days, just get rid of it).

**Comment by ghost** [comment]
Yes. Nothing will happen. They are totally random. I had similar experience but nothing happened.

**Comment by lestephane** [comment]
This is not only a problem for nasty newsletters.

If a spammer sends a message where hundreds of people are in the To, CC header, only one of them being one of my aliases, each of the recipients cause a reverse alias to be created behind my alias. This is polluting my reverse aliases list with doxed email addresses from other people that I never had any interaction with. This is a data privacy disaster.

At the very least, you must give a way to auto delete those reverse aliases if there has been 0 activity for a configurable period!
